### ISSUE TITLE
Unable to reproduce loss/accuracy result

### ISSUE THREAD

**Issue by gudgud96** [jason9693/MusicTransformer-tensorflow2.0#8]
Hi jason9693 and community,

Thanks for the wonderful implementation of Music Transformer! Really appreciate your work.

I have tried using the preset hyperparameter values to train the model, but unfortunately I can't get the same results as claimed in the graphs shown in README.md.

I hereby attach my loss and accuracy graphs -- it already converges at 5.0+ and 0.04+ during epoch 200, not as claimed to be 2.2+ and 0.34+ in the graph.

Loss:
![image](https://user-images.githubusercontent.com/13913915/66916072-119a8c80-f04d-11e9-91f3-8a693f6886d9.png)

Accuracy:
![image](https://user-images.githubusercontent.com/13913915/66916086-195a3100-f04d-11e9-8748-658fae0fa8d5.png)

Would appreciate any advice from this community on measures to take to reproduce the results. Thank you =)

**Comment by jason9693** [comment]
Hi, @gudgud96.
Actually I've found omitted updated learning rate value at README😂
I've just update learning rate value at readme now. 

thank you
